src/middlewared/middlewared/plugins/auth.py

```diff
@@ -166,44 +160,6 @@ def dump(self):
         }
 
 
-class TokenSessionManagerCredentials(SessionManagerCredentials):
```

Comment:
Now `flake8` says that the following imports are not used in this file:
- `SessionManagerCredentials`
- `List`
- `Ref`
- `TCPIPOrigin`

(I suppose it's _possible_ that we reexport them, but unlikely.)

Also, extra space on line 609:
```
                user_info =  (await middleware.call(
```